docs/_includes/ww-admin-tasks-content.md

```diff
@@ -0,0 +1,250 @@
+
+<span class="admin-task__permissions">Permissions required: User must be marked as staff; “Add/edit response templates” must be ticked..</span>
+
+Staff users with the appropriate permissions may create and edit pre-written responses, so that they can be applied to reports quickly and easily.
+
+<img alt="Checking property collection details" src="/assets/img/ww-user-guide/WasteWorks-response-templates.png" />
+
+#### Creating a template
+
+Click on ‘Templates’ in the admin menu. You will see a table of any existing templates. Click on the button below, marked ‘New template’.
+
+Common template responses deal with statuses such as: Report received; Collection scheduled; Container ordered; Report under investigation.
+
+There is no limit to the number of template responses you can create. When creating templates, you should consider:
+
+What are the most common response types that you send back to report makers? It may be worth examining recent responses to find out.
+How should you word your response template text to make it as easy as possible for the report maker to understand the status of their issue?
+What title should you give the template so that it can be easily identified from a drop-down list when it is being applied by staff users?
+Input a title and some text for your template, then set which categories it should apply to. You may create different templates for each category, or use the same templates across multiple categories.
+
+You can also set a State so that the template response is automatically used when that state is applied to a report.
+
+For example, you can set it so that when a staff user changes a report’s state to ‘No further action’, the ‘Resolved’ status update text is automatically applied. While this functionality can be a time-saver, we advise using it with caution to ensure that the template text is applicable to every situation in which is will be automatically applied.
+
+If you have an Open311 connection, you can click ‘auto-response’ so that a template will be applied when the state is updated by the automated Open311 process. In this instance, if your Open311 server returns extra text as part of the update, you may put the placeholder {{description}} in the template here, and that placeholder will be replaced by the text from the Open311 server.
+
+If you don’t have an Open311 connection, or your Open311 connection does not provide an immediate initial update, there is a special case where if a template is assigned to the Open state, and marked as ‘auto-response’, then it will automatically be added as a first update to any new report created that matches the template (ie. in the relevant category if assigned). This lets you give e.g. estimated timescales or other useful information up front.
+
+If you enter text in the ‘Text for email alert field’, the template text will update the report on the website and the email text will be sent to the user if they have opted into alerts.
+
```

Comment:
Templates needs an extra section explaining their second use to provide the lock out messages on bin days page to give better explanations than backend to why a bin wasn't collected etc.